### ISSUE TITLE
Alamofire 5 Documentation 

### ISSUE THREAD

**Issue by Mahan3340** [Alamofire/Alamofire#2748]
when will the documentation be available ? I have not been using this library for a long time and its hard to follow and I don't intend to use older versions and migrate to new one later , I would appreciate that if you could just simultaneously start writing the documentations for the new version, it will be more helpful even if it's partially available than when there nothing. Not all the issues or the questions here or on the stack overflow are related to v5 and those that are related, will not cover everything.

**Comment by jshier** [comment]
Updates to our usage documentation will come before the Alamofire 5 release. In the meantime, our [API documentation](https://alamofire.github.io/Alamofire/) is pretty up-to-date and you can ask questions on StackOverflow (and tag `alamofire`) or on our [Swift community forum](https://forums.swift.org/c/related-projects/alamofire).
